caper/cromwell_metadata.py

```diff
@@ -295,14 +303,22 @@ def gcp_monitor(
                 ...
             ]
         """
+        import pandas as pd
+
         result = []
         file_size_cache = {}
+        workflow_id = self.workflow_id
 
         def gcp_monitor_call(call_name, call, parent_call_names):
             nonlocal result
             nonlocal excluded_cols
             nonlocal stat_methods
             nonlocal file_size_cache
+            nonlocal workflow_id
+            nonlocal task_name
```

Comment:
I know it's not part of this PR, but in the future this should probably be refactored to avoid use of `nonlocal`, and extract out the `gcp_monitor_call` from within the function.

Reply:
Thanks, will be working on this for the next refactoring.